src/main/java/pl/piekoszek/gorskimatches/equation/EquationMathChecker.java

```diff
@@ -0,0 +1,34 @@
+package pl.piekoszek.gorskimatches.equation;
+
+import org.springframework.stereotype.Component;
+
+@Component
+class EquationMathChecker {
+
+
+    boolean isMathematicallyCorrect(String equation) {
+
+        char[] equationChar = equation.toCharArray();
+
+        int a = Integer.parseInt(String.valueOf(equationChar[0]));
```

Comment:
`int a = equationChar[0] - '0';` is another option

Reply:
When changed to that - id does not pass test.

Reply:
How did you change it?